verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java

```diff
@@ -23,11 +17,10 @@
 public class VerticalSeekBar extends RelativeLayout {
 
     LayoutInflater inflater;
-    View layoutView;
-    View verticalSeekBarBackground;
-    View verticalSeekBarThumb;
-    float dX = 0, dY = 0;
-    int marginTop = 0;
+    View layoutView, verticalSeekBarBackground, verticalSeekBarThumb;
+    VerticalSeekBarListener listener;
+    float dY = 0;
+    int marginTop = 0, valor = 0, step = 0, calculatedValue = 0;
```

Comment:
All properties have english names, but the `valor` is in portuguese. Can you fix that?

Reply:
valor has no english word... fixing :)